Foundation/NSSwiftRuntime.swift

```diff
@@ -14,10 +14,16 @@ import CoreFoundation
 // This mimics the behavior of the swift sdk overlay on Darwin
 #if os(OSX) || os(iOS)
 @_exported import Darwin
-#elseif os(Linux)
+#elseif os(Linux) //// || os(Android) // Causes ambiguity in NSScanner.swift
```

Comment:
You misunderstand. That code is fine in NSScanner as an internal extension. But it cannot be a public API in the stdlib.